lib/graphql.schema.js

```diff
@@ -137,6 +162,76 @@ const _getRelationshipFromDescription = relationship => {
   }
 };
 
+const _createEntity = entity => {
+  const entityInputType = new GraphQLInputObjectType({
+    name: `${entity.name}InputType`,
+    fields: () => (_getSchemaFieldsFromEntityWithoutRelationship(entity))
```

Comment:
`_getSchemaFieldsFromEntityWithoutRelationship` returns also id which should not be possible to specify when creating entity